scripts/validate.py

```diff
@@ -115,11 +94,21 @@ def run(global_config):
         manifest_file = manifest_file.strip()
         files = utils.get_fileinfo_list_from_s3_manifest(manifest_file)
         fail_list = []
+
+        logger.info("Beginning validation process")
+        file_number = 0
+        file_total = len(files)
         for fi in files:
+            file_number += 1
+            if file_number % 100 == 0:
+                logger.info(f"File {file_number}/{file_total}")
+
             del fi["url"]
             fi["aws_url"], fi["gs_url"], fi["indexd_url"] = None, None, None
 
-            fi["indexd_url"] = indexd_records.get(fi.get("id"), [])
+            print(utils.get_indexd_record(fi.get("id")))
+            fi["indexd_url"] = utils.get_indexd_record(fi.get("id"))
```

Comment:
I think this is fine for now but I would say after we're done fixing the aws command, make another PR to use asyncio. We can definitely get more speed from leveraging asyncio. Most of the things we do for these jobs are multi-threaded and this is a pretty time consuming part of the code that we can definitely speed up.